fiftyone/core/models.py

```
@@ -527,227 +518,116 @@ def _apply_image_model_data_loader(
 
-    samples = _select_fields_for_inference(samples, model, field_mapping)
-
-    with contextlib.ExitStack() as context:
-        pb = context.enter_context(
-            fou.ProgressBar(total=total_frame_count, progress=progress)
+    Only supports applying image model to videos frames.
+    """
+    if samples.media_type != fom.VIDEO:
+        raise fom.MediaTypeError(
+            f"Unsupported media type {samples.media_type}."
         )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)